site/src/pages/TemplateVersionEditorPage/PublishTemplateVersionDialog.tsx

```diff
@@ -10,11 +10,25 @@ import type { PublishVersionData } from "pages/TemplateVersionEditorPage/types";
 import type { FC } from "react";
 import { getFormHelpers } from "utils/formUtils";
 import * as Yup from "yup";
+import {
+	HelpTooltip,
+	HelpTooltipContent,
+	HelpTooltipLink,
+	HelpTooltipLinksGroup,
+	HelpTooltipText,
+	HelpTooltipTitle,
+	HelpTooltipTrigger,
+} from "../../components/HelpTooltip/HelpTooltip";
+import { docs } from "../../utils/docs";
 
 export const Language = {
 	versionNameLabel: "Version name",
 	messagePlaceholder: "Write a short message about the changes you made...",
-	defaultCheckboxLabel: "Promote to default version",
+	defaultCheckboxLabel: "Promote to active version",
+	activeVersionHelpTitle: "Active versions",
+	activeVersionHelpText:
+		"Templates can enforce that a version be used for all workspaces (enterprise-only)",
```

Comment:
This implies that any version can be enforced when it's only the active one.

```suggestion
		"Templates can enforce that the active version be used for all workspaces (enterprise-only)",
```